12-06-25-assignment-8/question-8.js

```diff
@@ -0,0 +1,27 @@
+// Find Error in below questions and rectify them
+
+function Person(name, age) {
+  this.name = name;
+  this.age = age;
+}
+Person.prototype.getDetails = function () {
+  console.log(this.name + " is " + this.age + " years old");
+};
+var person = new Person("John Doe", 30);
+delete person.name;
+person.getDetails();
```

Comment:
This code does not have any issue. it will still work. the output will be "undefined is 30 years old"